src/System/Relay/middleware/cacheHeaderMiddleware.ts

```diff
@@ -34,6 +41,9 @@ export const shouldSkipCDNCache = (req, user, foundRoute) => {
   if (isLoggedIn) {
     return true
   }
+
+  // If none of the above conditions are met, we cache the request.
+  return false
```

Comment:
This doesn't change anything as this method returned undefined (and so was fine when used in truth-y checks).

But - probably better to be explicit, esp as we should add some better typings to this file